draft-tus-resumable-uploads-protocol.md

```diff
@@ -248,7 +248,7 @@ The request MUST use the `HEAD` method and include the `Upload-Token` header. Th
 
 The client MUST NOT perform the Offset Retrieving Procedure ({{offset-retrieving}}) while the Upload Transfer Procedures ({{upload-transfer}}) is in progress.
 
-If the server considers the upload associated with this token active, it MUST send back a `204 (No Content)` response with a header `Upload-Offset` which indicates the resumption offset for the client.
+If the server has resources allocated for this token, it MUST send back a `204 (No Content)` response. The response MUST include the `Upload-Offset` header set to the current resumption offset for the client. The response MUST include the `Upload-Incomplete` header which is set to true if and only if the upload is incomplete. An upload is considered complete if and only if the server completely and succesfully received a corresponding Upload Transfer Procedure  ({{upload-transfer}}) request with a falsy `Upload-Incomplete` header.
```

Comment:
>If the server considers the upload associated with this token active, 

Should probably still be there from #26

Reply:
"falsy" is a bit vague

Reply:
Add brief explanation that the server may deactivate the token immediately after the transfer completes, so it won't ever respond with `Upload-Incomplete: ?0`. We are not forcing servers to keep the completed upload around.

Reply:
Thank you all for the feedback!